components/Gallery/GalleryItem.tsx

```diff
@@ -12,11 +12,12 @@ export const GalleryItem = ({ source, title }: Props) => {
       <Image
         src={source}
         alt={title}
-        width="100%"
-        height="100%"
         sizes="(max-width: 767px) 30vw, (max-width: 1200px) 25vw, 17vw"
-        objectFit="cover"
-        layout="responsive"
+        style={{
```

Comment:
something is broken - items should have the same height (should be more in rectangle shape)
<img width="1433" alt="Screenshot 2022-12-11 at 13 10 25" src="https://user-images.githubusercontent.com/24607370/206902761-3d163176-8ad5-49f0-90a2-12e867736127.png">